### PR TITLE
Smooth scrolling support

### PR DESCRIPTION
**Pull request by lenemter**

### CODE REVIEW

File: src/MainView.vala [hunk 1]
```diff
@@ -342,4 +362,67 @@ public class Power.MainView : Gtk.Grid {
             scale.value_changed.connect (on_scale_value_changed);
         }
     }
+
+    /* Handles both SMOOTH and non-SMOOTH events.
+     * In order to deliver smooth volume changes it:
```
Comment (jeremypw) [inline src/MainView.vala:367]:
```suggestion
     * In order to deliver smooth brightness changes it:
```

**Review by jeremypw — CHANGES_REQUESTED** [review]
This works as designed but I think Dani's opinion should be sought on the correct brightness step.  I think it should be smaller as the eyes are more sensitive to brightness changes than the ears are to volume changes and a step of 5.0 seems rather jerky to me.  I would suggest soemwhere between 1 and 2?

**Comment by lenemter** [comment]
@jeremypw Wingpanel indicator's step is exactly 5.0, that's why I used it

**Comment by jeremypw** [comment]
Yes, I just realized that - so I will approve this and then both the switchboard and wingpanel plugs can be changed together if required.

**Review by jeremypw — APPROVED** [review]

**Pull request merged**